lib/inferno/apps/web/index.html.erb

```diff
@@ -29,7 +31,12 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
+    <div class='wrapper'>
+      <% if File.exist? (File.join(Inferno::Application.root, 'banner.html.erb')) %>
```

Comment:
This won't work in test kits. I think the banner needs to go in `config`, and then the path would be `File.join(Dir.pwd, 'config', 'banner.html.erb')`.

Reply:
@Jack-Fraser16 can you update it to look here so we can get this in?